tools/run_local_ironic.sh

```diff
@@ -170,10 +170,15 @@ fi
 
 BASIC_AUTH_MOUNTS=""
 IRONIC_HTPASSWD=""
+IRONIC_HTPASSWD_FILE="${IRONIC_DATA_DIR}/auth/ironic-htpasswd"
+IRONIC_HTPASSWD_MOUNT=""
+set +x
 if [ -n "$IRONIC_USERNAME" ]; then
      IRONIC_HTPASSWD="$(htpasswd -n -b -B "${IRONIC_USERNAME}" "${IRONIC_PASSWORD}")"
-     IRONIC_HTPASSWD="--env HTTP_BASIC_HTPASSWD=${IRONIC_HTPASSWD} --env IRONIC_HTPASSWD=${IRONIC_HTPASSWD}"
+     echo "${IRONIC_HTPASSWD}" > "${IRONIC_HTPASSWD_FILE}"
```

Comment:
you could probably just redirect the output of the htpasswd command directly into the file and remove the need of the IRONIC_HTPASSWD variable entirely

Reply:
Removed the variable